tests/tests_suite_test.go

```diff
@@ -104,7 +105,12 @@ var _ = SynchronizedBeforeSuite(testsuite.SynchronizedBeforeTestSetup, testsuite
 var _ = SynchronizedAfterSuite(testsuite.AfterTestSuiteCleanup, testsuite.SynchronizedAfterTestSuiteCleanup)
 
 var _ = AfterEach(func() {
-	testCleanup()
+	skipGlobalCleanupLabelText := decorators.SkipGlobalCleanup[0]
+
+	skipGlobalCleanup, err := CurrentSpecReport().MatchesLabelFilter(skipGlobalCleanupLabelText)
+	if !skipGlobalCleanup || err != nil {
```

Comment:
I would expect an error to not occur here, not silently proceed

Reply:
`Expect(err).ToNot(HaveOccurred())` should be fine

Reply:
You're in an aftereach, should be fine

Reply:
This aftereach is no different to any other one found in any test suite

Reply:
Anyway you can just panic() if you want it's not a big deal, just no reason for this to ever error